docker/Dockerfile.vineyard-llm-python-dev

```diff
@@ -0,0 +1,215 @@
+COPY modules/llm-cache /work/v6d/modules/llm-cache
+COPY thirdparty /work/v6d/thirdparty
+
+COPY python/client.cc /work/v6d/python/client.cc
+COPY python/core.cc /work/v6d/python/core.cc
+COPY python/error.cc /work/v6d/python/error.cc
+COPY python/pybind11_docs.cc /work/v6d/python/pybind11_docs.cc
+COPY python/pybind11_docs.h /work/v6d/python/pybind11_docs.h
+COPY python/pybind11_utils.cc /work/v6d/python/pybind11_utils.cc
+COPY python/pybind11_utils.h /work/v6d/python/pybind11_utils.h
+COPY python/vineyard.cc /work/v6d/python/vineyard.cc
+
+COPY python/vineyard /work/v6d/python/vineyard
+
+RUN cd /work/v6d && \
+    mkdir build && \
+    cd build && \
+    export PATH=$PATH:/usr/local/bin && \
+    pip3 install -U pip setuptools wheel && \
+    cmake .. -DBUILD_SHARED_LIBS=OFF \
+             -DBUILD_VINEYARD_SERVER=OFF \
+             -DBUILD_VINEYARD_CLIENT=ON \
+             -DBUILD_VINEYARD_PYTHON_BINDINGS=ON \
+             -DBUILD_VINEYARD_PYPI_PACKAGES=OFF \
+             -DBUILD_VINEYARD_LLM_CACHE=ON \
+             -DBUILD_VINEYARD_BASIC=OFF \
+             -DBUILD_VINEYARD_IO=OFF \
+             -DBUILD_VINEYARD_GRAPH=OFF \
+             -DBUILD_VINEYARD_MALLOC=OFF \
+             -DBUILD_VINEYARD_MIGRATION=ON \
+             -DBUILD_VINEYARD_HOSSEINMOEIN_DATAFRAME=OFF \
+             -DBUILD_VINEYARD_TESTS=OFF \
+             -DBUILD_VINEYARD_TESTS_ALL=OFF \
+             -DBUILD_VINEYARD_BENCHMARKS=OFF \
+             -DBUILD_VINEYARD_BENCHMARKS_ALL=OFF \
+             -DBUILD_VINEYARD_COVERAGE=OFF \
+             -DBUILD_VINEYARD_PROFILING=OFF \
+             -DCMAKE_BUILD_TYPE=Release \
+             -DPYTHON_EXECUTABLE=/usr/local/bin/python && \
+    make -j$(nproc) && \
+    make vineyard_client_python -j$(nproc)
+
+COPY README.rst /work/v6d/README.rst
+COPY setup.py /work/v6d/setup.py
+COPY setup_bdist.py /work/v6d/setup_bdist.py
+COPY setup_io.py /work/v6d/setup_io.py
+COPY setup_llm.py /work/v6d/setup_llm.py
+COPY requirements.txt /work/v6d/requirements.txt
+COPY requirements-dev.txt /work/v6d/requirements-dev.txt
+COPY requirements-extra.txt /work/v6d/requirements-extra.txt
+COPY requirements-kubernetes.txt /work/v6d/requirements-kubernetes.txt
+COPY requirements-setup.txt /work/v6d/requirements-setup.txt
+COPY modules/io /work/v6d/modules/io
+
+# make sure the library been copied
+RUN cd /work/v6d/build && \
+    make vineyard_client_python -j$(nproc) && \
+    cd /work && \
+    export LD_LIBRARY_PATH=/work/v6d/build/lib:$LD_LIBRARY_PATH && \
+    mkdir -p fixed_wheels/ && \
+    cd /work/v6d && \
+    rm -rf dist build/lib* && \
+    /usr/local/bin/python setup_llm.py bdist_wheel && \
+    mv /work/v6d/dist/*.whl /work/fixed_wheels/ && \
+    rm -rf /work/v6d
+
+
+# generate final image
+FROM python:3.10-slim-buster
+
+SHELL ["/bin/bash", "-l", "-c"]
+COPY --from=wheel /work/fixed_wheels/*.whl /tmp/
+
+RUN cd /tmp &&\
+    pip3 install --no-cache-dir *.whl ipython
+
+ENTRYPOINT ["/bin/bash", "-l", "-c" ]
```

Comment:
Remove this file. I think current `vineyard-python-dev` would be enough as we no longer use the `renameat2` API.

Reply:
Make sense. Thanks for pointing it out.